platform_versioned_docs/version-24.1.0/enterprise/_templates/docker/tower.env

```
@@ -34,6 +34,5 @@ GROUNDSWELL_SERVER_URL="http://groundswell:8090"
 #TOWER_DATA_EXPLORER_CLOUD_DISABLED_WORKSPACES=
 #
 #TOWER_DATA_STUDIO_CONNECT_URL=<YOUR CONNECT URL, SAME AS `CONNECT_PROXY_URL` IN `data-studios.env` >
-#TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN=<YOUR INITIAL ACCESS TOKEN, IDEANTICAL AS `INITIAL_ACCESS_TOKEN` in `data-studios.env`>
+#TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN=<YOUR INITIAL ACCESS TOKEN, IDENTICAL AS `INITIAL_ACCESS_TOKEN` in `data-studios.env`>
```

Changed variable name from  to `INITIAL_ACCESS_TOKEN` to `OIDC_CLIENT_REGISTRATION_TOKEN`, for better understanding

---

```suggestion
#TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN=<YOUR INITIAL ACCESS TOKEN, IDENTICAL TO `OIDC_CLIENT_REGISTRATION_TOKEN` in `data-studios.env`>
```

@endre-seqera, is this suggestion correct? It's a little bit confusing to me. Should it not just be: 

#TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN=<use the same `OIDC_CLIENT_REGISTRATION_TOKEN` as in `data-studios.env`>

---

You are right, the latter suggestion is easier to understand, we should go with it:
```
#TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN=<use the same OIDC_CLIENT_REGISTRATION_TOKEN as in data-studios.env>
```